### DISCUSSION TITLE
Scanning Comments Faster

### DISCUSSION THREAD

**Discussion by Lightning11wins** [ThioJoe/YT-Spammer-Purge#985] [Q&A] [answered]
I have very fast WIFI, as well as a decent CPU, GPU, and plenty of memory available. Since very few of these resources are being used by the spam purge. I was wondering: where is the bottleneck for scanning speed? Is it the YouTube API?

![slow](https://user-images.githubusercontent.com/42805582/205959727-72fb82a4-ed36-440d-beeb-ba067f918367.png)

**Comment by ThioJoe** [comment, marked as answer]
Yea by far the biggest bottleneck is waiting for the API responses

**Reply by Lightning11wins** [reply]
So it's just that YouTube doesn't want to go over a certain speed. That makes sense.
